lib/openglvis.cpp

```diff
@@ -261,14 +261,13 @@ ::DrawTriangle(gl3::GlDrawable& buff,
       return;
    }
 
-   std::array<float, 2> texcoord[3];
+   float texcoord[3];
    std::array<float, 3> fpts[3];
    std::array<float, 3> fnorm = {(float) nor[0], (float) nor[1], (float) nor[2]};
 
    for (int i = 0; i < 3; i++)
```

Comment:
(minor) I don't think this will every change but consider using some constexpr instead of `3`